lusidtools/pandas_utils/lusid_pandas.py

```diff
@@ -16,6 +16,8 @@ def lusid_response_to_data_frame(lusid_response, rename_properties: bool = False
     :param lusid_response: a response from the LUSID APIs (e.g. VersionedResourceListOfPortfolioHolding)
     :param bool rename_properties: this parameter formats the returned DataFrame. Specifically, the formatter does two things;
     (1) removes any metadata columns for properties and SHKs, and (2) simplifies the naming of column headers for properties and SHKs.
+    :param dict rename_mapping: a dictionary which is used to map old column headers to new ones. The dictionary key is old header
+    while the dictionary value in new header.
```

Comment:
add a comment saying what happens if the given property name doesn't exist

Reply:
Sure, done.